app/Http/Controllers/NavaidController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Navigation\Navaid;
+use Illuminate\Http\JsonResponse;
+
+class NavaidController
+{
+    public function getNavaidsDependency(): JsonResponse
```

Comment:
Are there future methods planned for this? Otherwise it could just be an invokable

Reply:
Changed